sdk/servicebus/azure-servicebus/api_samples.py

```diff
@@ -0,0 +1,219 @@
+## Sample Code:
+
+### creation of different entity clients
+
+# Top Level ServiceBusClient
+sb_client = ServiceBusClient.from_connection_string(conn_str="conn_str")
+sb_client = ServiceBusClient(
+    fully_qualified_namespace="fully_qualified_namespace",
+    crendetial=ServiceBusSharedKeyCredential("policy", "key")
+)
+
+queue_sender = sb_client.get_queue_sender(queue_name="queue_name")
+queue_receiver = sb_client.get_queue_receiver(queue_name="queue_name")
+topic_sender = sb_client.get_topic_sender(topic_name="topic_name")
+subscription_receiver = sb_client.get_subscription_receiver(
+    topic_name="topic_name",
+    subscription_name="subscription_name"
+)
+
+# Queue Sender
+queue_sender = ServiceBusSender.from_connection_string(
+    conn_str="conn_str",
+    queue_name="queue_name"
+)
+
+queue_sender = ServiceBusSender(
+    fully_qualified_namespace="fully_qualified_namespace",
+    queue_name="queue_name",
+    ServiceBusSharedKeyCredential("policy", "key")
+)
+
+# Queue Receiver
+queue_receiver = ServiceBusReceiver.from_connection_string(
+    conn_str="conn_str",
+    queue_name="queue_name"
+)
+
+queue_receiver = ServiceBusReceiver(
+    fully_qualified_namespace="fully_qualified_namespace",
+    queue_name="queue_name",
+    credential=ServiceBusSharedKeyCredential("policy", "key"),
+    session_id="test_session"
+)
+
+# Topic Sender
+topic_sender = ServiceBusSender.from_connection_string(
+    conn_str="conn_str",
+    topic_name="topic_name"
+)
+
+topic_sender = ServiceBusSender(
+    fully_qualified_namespace="fully_qualified_namespace",
+    topic_name="topic_name",
+    credential=ServiceBusSharedKeyCredential("policy", "key")
+)
+
+# Subscription Receiver
+subscription_receiver = ServiceBusReceiver.from_connection_string(
+    conn_str="conn_str",
+    topic_name="topic_name",
+    subscription_name="subscription_name"
+)
+
+subscription_receiver = ServiceBusReceiver(
+    fully_qualified_namespace="fully_qualified_namespace",
+    topic_name="topic_name",
+    subscription_name="subcription_name",
+    credential=ServiceBusSharedKeyCredential("policy", "key"),
+    session_id="test_session"
+)
+
+
+### Send and receive
+# 1 Send
+sb_client = ServiceBusClient.from_connection_string(conn_str="conn_str")
+queue_sender = sb_client.get_queue_sender(queue_name="queue_name")
+
+with sb_client:
+    with queue_sender:
+        # 1.1 send single
+        message = Message("Test message")
+        queue_sender.send(message)
+
+        # 1.2 send batch
+        batch_message = topic_sender.create_batch(max_size_in_bytes=256 * 1204)
+        while True:
+            try:
+                batch_message.add(Message("Test message"))
+            except ValueError:
+                break
+
+        topic_sender.send(batch_message)
+
+        # 1.3 schedule
+        schedule_message = Message("scheduled_message")
```

Comment:
What kind of body types do you support out of the box? Do you default to any JSON serialization, etc...?

Reply:
we accept bytes or string